README.md

```diff
@@ -231,6 +234,9 @@ Since `v1.1.0` if you're going to edit a release and an asset name in that relea
   * ### `isPrerelease` <sup>since v1.0.7</sup>
     Set this value to `true` if you want this release to be flagged as a prerelease.
 
+  * ### `update_assets` <sup>since v1.1.2</sup>
+    Set this value to `true` if you want to be able to be able to replace assets that already exist in the release, otherwise they will be considered errors.
```

Comment:
Duplicated `to be able`
```suggestion
    Set this value to `true` if you want to be able to replace assets that already exist in the release, otherwise they will be considered errors.
```